src/post_profile_point.ts

```
@@ -0,0 +1,51 @@
+import { DynamoDB, PutItemCommand } from '@aws-sdk/client-dynamodb';
+
+const dynamodb = new DynamoDB({});
+const TABLENAME = process.env.TABLENAME || "";
```

TABLENAMEが解決できないとdynamoをcallしたときにコケてしまうので、空文字にするのではなくここでfail fastさせても良い気がしました（環境変数指定がない限り、そもそも起動させない）

---

なるほど、handlerを渡すまえにfail fastというのはlambda的にNGなんですね。。。

> 処理を追加しました。

良いと思います！